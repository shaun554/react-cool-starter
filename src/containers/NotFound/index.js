import React from 'react';
import Helmet from 'react-helmet';

import styles from './styles.scss';

export default () => (
  <div className={styles.NotFound}>
    <Helmet title="Oops" />
    <p>Oops, Page was not found!</p>
  </div>
);
