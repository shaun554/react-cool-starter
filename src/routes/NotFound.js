import NotFound from '../containers/NotFound';

export default {
  path: '*',
  component: NotFound,
};
