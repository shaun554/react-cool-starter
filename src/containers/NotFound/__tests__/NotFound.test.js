import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../index';

describe('<NotFound />', () => {
  test('renders 404 error message', () => {
    const wrapper = shallow(<NotFound />);

    expect(wrapper.find('p').text()).toBe('Oops, Page was not found!');
  });
});
