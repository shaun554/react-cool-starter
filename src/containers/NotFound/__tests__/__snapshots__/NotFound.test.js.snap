// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders 1`] = `
<div
  className={undefined}
>
  <p>
    Oops, Page was not found!
  </p>
</div>
`;
